crates/trie/sparse/src/state.rs

```diff
@@ -102,75 +102,6 @@ impl<F: BlindedProviderFactory> SparseStateTrie<F> {
         self.storages.get_mut(account).and_then(|e| e.as_revealed_mut())
     }
 
-    /// Reveal unknown trie paths from provided leaf path and its proof for the account.
-    /// NOTE: This method does not extensively validate the proof.
-    pub fn reveal_account(
```

Comment:
i'd prefer to keep this methods, they are still useful. mb with `debug_assert!(!self.updates_enabled)`

Reply:
fair, useful at least for tests